default/scripting/species/common/planet_shields.macros

```diff
@@ -9,12 +9,24 @@ NATIVE_PLANETARY_SHIELDS
             effects = SetMaxShield value = Value + 10 accountinglabel = "NATIVE_PLANETARY_SHIELDS_LABEL"
 
     [[PROTECTION_FOCUS_SHIELDS]]
+    [[STANDARD_SHIELD_GROWTH]]
 '''
 
 
 AVERAGE_PLANETARY_SHIELDS
-'''[[PROTECTION_FOCUS_SHIELDS]]'''
+'''[[PROTECTION_FOCUS_SHIELDS]]
+[[STANDARD_SHIELD_GROWTH]]'''
 
+STANDARD_SHIELD_GROWTH
+'''        EffectsGroup // increase 1 per turn, up to max
+            scope = Source
+            activation = And [
+                Planet
+                (Source.LastTurnConquered < CurrentTurn)
+            ]
+            priority = [[AFTER_ALL_TARGET_MAX_METERS_PRIORITY]]
+            effects = SetShield value = min(Value(Target.MaxShield), Value + 1)
```

Comment:
Not gated by combat, keeping populated planets at 1.0 shields.
When not attacked, populated planets gain from both this and the effect added to defense.
Might move the macros with multiple entry points to scripting/common/meters_gain.macros, with a stacking group.